.env.sample

```diff
@@ -10,3 +10,9 @@ MOCK_SERVER_BASE_URL=http://localhost:3500
 
 # Ethirium Chain ID for reading contract data using Network Connector
 REACT_APP_NETWORK_CONNECTOR_CHAIN_ID=4
+
+# Mixpanel API key
```

Comment:
I was thinking we don't need default for these vars in the sample because devs usually wouldn't need to run analytics locally.

If `REACT_APP_MIXPANEL_TOKEN` and `REACT_APP_ANALYTICS_API_URL` are defined it'll try to log the event and sends a network request to to `ourApi/api.bingbonglyrics.co` lmao

could pollute logs if the api base url is pointed to dev or prod api

Reply:
Or we could change the switch for enabling analytics to something else

Reply:
lol truuu. okay makes sense, should take that out!